README.md

```diff
@@ -1,4 +1,4 @@
-# <div align="center"><a href="https://calculator.yoshitha.tk/"><b><i>WELCOME TO MY CALCULATOR WEBSITE</i></b></a></div>
+# <div align="center"><a href="https://calculator.yoshitha.tk/"><b><i>WELCOME!, TO MY NEW OFFICIAL WEBSITE</i></b></a></div>
```

Comment:
Sorry I forgot to tell you don't remove the CALCULATOR part only add that !, part @Mercysticks